src/components/Molecules/Footer/Nav/Nav.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import Text from '../../../Atoms/Text/Text';
+import { sizes } from '../../../../theme/shared/breakpoint';
+
+import {
+  Nav,
+  NavMenu,
+  NavItem,
+  NavLink,
+  SubNavMenu,
+  SubNavItem,
+  SubNavLink
+} from './Nav.style';
+
+const FooterNav = ({ navItems }) => {
+  const { menuGroup } = navItems;
+  const [isExpandable] = useState(false);
+  const [isSubMenuOpen, setIsSubMenuOpen] = useState({});
+
+  // Detect browser width size
+  const isSmallBreakpoint = window.innerWidth < sizes.small;
+
+  /**
+   * Always stop the main 'parent' link from actually firing, but do the
+   * collapsing for SM-MD breakpoints
+   */
+  const toggleSubMenu = item => event => {
+    event.preventDefault();
+
+    // Only run on SM, as we're only using the hide-show logic on SM
+    if (isSmallBreakpoint) {
+      setIsSubMenuOpen({ [item]: !isSubMenuOpen[item] });
+    }
+  };
+
+  return (
+    <Nav
+      aria-labelledby="main-menu"
+      isExpandable={isExpandable}
+      role="navigation"
+    >
+      <Text id="footer-menu" tag="h2">
+        Footer navigation
+      </Text>
+
+      {/* First level of the navigation (ul tag): Parent */}
+      <NavMenu role="menubar">
+        {menuGroup.map((group, index) => (
+          <NavItem
+            role="none"
+            key={group.id}
+            index={index}
+            isSubMenuOpen={!!isSubMenuOpen[group.id]}
+          >
+            {!isSmallBreakpoint ? (
+              <NavLink
+                href={group.url}
+                inline
+                aria-haspopup="true"
+                onClick={toggleSubMenu(group.id)}
+              >
+                <Text>{group.title}</Text>
+              </NavLink>
+            ) : (
+              <NavLink
+                href={group.url}
+                inline
+                aria-expanded={!!isSubMenuOpen[group.id]}
+                aria-haspopup="true"
+                onClick={toggleSubMenu(group.id)}
+              >
+                <Text>{group.title}</Text>
+              </NavLink>
+            )}
+            {/* Second level of the navigation (ul tag): Child(ren) */}
+            {group.links && group.links.length > 0 && (
+              <SubNavMenu
+                role="menu"
+                aria-label={group.title}
```

Comment:
I don't think we need `aria-label` here. It will read "Title" twice.

Reply:
This is taken from Louis' header work; will update both

Reply:
I've checked with 2 screenreaders and this is fine- i'd rather not remove this here. Will have a chat with Louis and have a look at it's usage within the next PR that'll cover the social link stuff